tests/benchmark_test_inputs/AD_grid_diesel/csv_elements/energyConversion.csv

```diff
@@ -0,0 +1,16 @@
+,unit,transformer_station_in,diesel_generator
+age_installed,year,5,5
+development_costs,currency,0,60000
+specific_costs,currency/kW,0,0
+efficiency,factor,0.96,0.34
+inflow_direction,str,Electricity (DSO),Electricity (diesel)
+installedCap,kW,1250,500
+label,str,Transformer station (DSO) consumption, Diesel generator
+lifetime,year,30,30
+specific_costs_om,currency/kW/year,0,38
+dispatch_price,currency/kWh,0,0.35
```

Comment:
Why does it both `specific_cost_om `and `dispatch_price` in the same range? Is this wear costs as well as fuel cost or is if fuel cost twice?

Reply:
I don't understand your question. The specific costs are 38 currency/kWh while the dispatch price is 0.35 currency/kWh. Which range are you referring to?

Reply:
Oh, I am sorry, did overread the point there. But, is 38 currency per supplied kWh not a bit much?

Reply:
I tried to look for prices.. it's a 500 kW generator. it's a bit tricky to find exact numbers. 
In any case, do we have a conclusion on the diesel generator? should I define it as a transformer or a source?